Examples/antsRegistration.cxx

```diff
@@ -193,7 +193,6 @@ antsRegistrationInitializeCommandLineOptions(itk::ants::CommandLineParser * pars
     option->SetShortName('z');
     option->SetUsageOption(0, "(1)/0");
     option->SetDescription(description);
-    option->AddFunction(std::string("1"));
```

Comment:
I don't think this should have been removed, I get a seg fault now unless I specify `--collapse-output-transforms 1`. The argument handling is inconsistent and probably needs to be refactored, but some args (like this) have the convention that a default is assumed. In others, there is a check for the number of functions and only if > 0 is the value queried.

Reply:
I this going to impact ANTsR and ANTsPy?  I believe these do set the option internally.